src/vxingest/main.py

```diff
@@ -202,37 +206,98 @@ def create_dirs(paths: list[Path]) -> None:
         path.mkdir(parents=True, exist_ok=True)
 
 
-class JobDoc(TypedDict):
-    """A type class describing the expected return result from Couchbase"""
+class JobRunCriterion(TypedDict):
+    """A type class describing the criterion for running a job, and the
+    expected return result from Couchbase"""
 
     id: str
     name: str
     offset_minutes: int
     run_priority: int
-    sub_type: str
+    subType: str
```

Comment:
A smaller thing - since this is python, they prefer snake_case for names. Since we're between Go & Python for our couchbase clients we'll have some inconsistencies between the code variable name & the database variable name. But I think it's good to follow language conventions.